foam_wrapper/version.py

```diff
@@ -0,0 +1 @@
+version = '0.0.1dev'
```

Comment:
The `version.py` files is an automatically generate file and should not be added to the repository